src/Native/Bootstrap/platform.unix.cpp

```diff
@@ -27,7 +27,7 @@ int32_t WideCharToMultiByte(uint32_t CodePage, uint32_t dwFlags, uint16_t* lpWid
     throw 42;
 }
 
-int32_t MultiByteToWideChar(uint32_t CodePage, uint32_t dwFlags, const uint8_t * lpMultiByteStr, int32_t cbMultiByte, uint16_t* lpWideCharStr, int32_t cchWideChar)
+extern "C" int32_t MultiByteToWideChar(uint32_t CodePage, uint32_t dwFlags, const uint8_t * lpMultiByteStr, int32_t cbMultiByte, uint16_t* lpWideCharStr, int32_t cchWideChar)
```

Comment:
The `extern "C"` should be on all functions in this file, except for `UTF8ToWideCharLen` and `UTF8ToWideChar`.